examples/src/scriptsTest/scala/ai/h2o/sparkling/ScriptsTestSuite.scala

```diff
@@ -157,21 +157,6 @@ class CityBikeSharing extends ScriptsTestHelper {
   }
 }
 
-@RunWith(classOf[JUnitRunner])
```

Comment:
This script is almost identical to AirlinesWithWeatherDemo -> no point in keeping duplicate code and testing the same code twice ( it just predicts arr delay, the demo predicts dep delay)